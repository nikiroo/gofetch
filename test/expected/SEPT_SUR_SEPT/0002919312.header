0Pour la première fois, Trump se montre favorable à une solution à deux Etats	null/SEPT_SUR_SEPT/0002919312		70
i  [Israel-Palestine] 26-09-2018 16:46
i
i    Le  président  américain  Donald Trump s%27est déclaré mercredi
i    favorable  pour  la  première  fois à une solution à deux Etats
i    dans le conflit ...
i
