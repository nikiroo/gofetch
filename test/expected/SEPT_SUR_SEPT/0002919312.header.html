<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h1><a href='0002919312.html'>Pour la première fois, Trump se montre favorable à une solution à deux Etats</a></h1>
	<div class='details'>([Israel-Palestine] 26-09-2018 16:46)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		Le président américain Donald Trump s%27est déclaré mercredi favorable pour la première fois à une solution à deux Etats dans le conflit ...
	</div>
<hr/>
</div>
</body>
